src/qml/filters/audiolevelgraph/vui.qml

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2017-2020 Meltytech, LLC
```

Comment:
Change copyright year to current year only in all new files even when copying and pasting a lot of the code from somewhere else.

Reply:
done